### ISSUE TITLE
Spoonbill egg not in day 2 location

### ISSUE THREAD

**Issue by TheBlackWitch** [jeanropke/RDR2CollectorsMap#398]
The egg is not showing up in the tree where it is marked. Tried restarting several times. A new location maybe..?

**Comment by jeanropke** [comment]
Hey. Probably it's a game bug. I get this egg last night (02:00 GMT0)

**Comment by AdamNortonUK** [comment]
The eggs in trees are the most buggy collection items, they either don’t show up in the tree but you can hear the collectible in dead eye but not see the gold particles, or you can see the gold particles but there is no nests, but sometimes you can see the egg floating where the nest would be, so you can sometimes just shot the egg down.

The other bugs are that the eggs fall down under the map and cannot be collected.

Long story short, if eggs don’t show up, it’s a game bug. I’ve use the map extensively on all day cycles and items are where the map show that they are, it’s just the game being buggy if they do not show up in the game.
